### ISSUE TITLE
Achievements option for game achievements

### ISSUE THREAD

**Issue by crazyaccess** [Riku32/DiscordLink#8]
I think an achievement toggle should be added so that when a player gets an achievement ingame then it should update either the sync channel in discord or || optional : have a separate channel for the achievement listing all the achievements made by that particular play ||

**Comment by JSH32** [comment]
If anything this should just automatically happen if `chat.enabled` is enabled and the server is set to show advancements. I'll add this tomorrow hopefully. It will show in the linked channel similar to join/leave/death messages

**Comment by crazyaccess** [comment]
Aight or i might help...

**Comment by JSH32** [comment]
Yeah sorry about not being able to help much, currently went on a trip lmao. Hopefully, we add all this shit and ill be able to make a new release.

**Comment by crazyaccess** [comment]
Hmmm i am also thinking of cross chat pings ?

**Comment by crazyaccess** [comment]
Like if both the discord and the minecraft accounts are linked then they can cause crosslinked pings

**Comment by JSH32** [comment]
Ehhh, I don't think its a good idea but I might be able to do something like pinging a user by their in game username if they are online will ping them in game and make a noise. If they arent then ping them in discord? But then again maybe not. Make another issue for this one

**Comment by JSH32** [comment]
The main reason I dont do cross chat pings is because of risk of @ everyone. Which is why I put a space in between it

**Comment by crazyaccess** [comment]
> The main reason I dont do cross chat pings is because of risk of @ everyone. Which is why I put a space in between it

I mean you can always have toggle's for that
and a simple check for that won't be that hard
